src/lib/sitemap/docs-content-fields.ts

```diff
@@ -6,14 +6,45 @@
 import { makeSitemapField } from './helpers'
 
 export async function allDocsFields() {
-	const getDocsPaths = await fetch(
+	const getContentAPIDocsPaths = await fetch(
 		`${process.env.MKTG_CONTENT_DOCS_API}/api/all-docs-paths`
 	)
-	const { result: docsResult } = await getDocsPaths.json()
-	return docsResult.map((page: { path: string; created_at: string }) =>
-		makeSitemapField({
-			slug: `${page.path}`,
-			lastmodDate: page.created_at,
-		})
+	const { result: contentAPIDocsResult } = await getContentAPIDocsPaths.json()
+
+	// If there are docs that have been migrated to the unified docs repo, get the paths for those docs
+	// and merge them with the paths for the docs that haven't been migrated from the content API
+	if (__config.flags?.unified_docs_migrated_repos.length > 0) {
```

Comment:
@LeahMarieBush can you also add a check if we are in the using the unified-docs-sandbox?

`process.env.HASHI_ENV === 'unified-docs-sandbox'`